packages/vite-plugins/raw-imports/src/plugin.ts

```diff
@@ -0,0 +1,118 @@
+import type { Plugin } from 'vite';
+import path from 'node:path';
+import { readFileSync } from 'node:fs';
+
+/**
+ * Options for configuring the raw imports plugin.
+ */
+export interface RawImportsPluginOptions {
+  /**
+   * File extensions to handle with this plugin.
+   * Files with other extensions will be handled by Vite's built-in `?raw` support.
+   *
+   * @default ['md']
```

Comment:
Interface doesn't have any defaults. remove default in interface and add as comment on `rawImportsPlugin`